test/3x/torch/quantization/weight_only/test_gptq.py

```diff
@@ -32,17 +43,21 @@ def test_accuracy_improvement(self):
         # test_default_rtn_config
         model = copy.deepcopy(self.tiny_gptj)
         quant_config = get_default_rtn_config()
-        model = quantize(model, quant_config, run_fn=run_fn)
+        model = prepare(model, quant_config)
+        run_fn_for_rtn(model)
+        model = convert(model)
         rtn_label = model(self.example_inputs)[0]
         rtn_atol = (rtn_label - self.label).amax()
         # test_default_gptq_config
         model = copy.deepcopy(self.tiny_gptj)
         quant_config = get_default_gptq_config()
-        model = quantize(model, quant_config, run_fn=run_fn)
```

Comment:
As we want to maintain `quantize` for internal use, how about keeping the UTs for it and adding new UTs for `prepare`/`convert`?

Reply:
I don't think we want to maintain quantize for internal use. It's deprecated and not suggested.